src/components/component-playground.jsx

```diff
@@ -525,5 +548,40 @@ module.exports = React.createClass({
       orientation: contentNode.offsetHeight > contentNode.offsetWidth ?
                    'portrait' : 'landscape'
     });
+  },
+
+  _getFilteredComponents() {
```

Comment:
Judging by the code, this methods actually returns the fixtures that match the needle while preserving the `<ComponentName, ComponentDetails>` structure. How about moving [this logic](https://github.com/uberVU/react-component-playground/blob/6486b97590775795c73d00b6286b22104224e868/src/components/component-playground.jsx#L568-L573) here and renaming the function to `getComponentsWithMatchingFixtures`?